src/main/java/hudson/plugins/git/GitChangeSet.java

```diff
@@ -356,7 +357,21 @@ String getParentCommit() {
      * @param createAccountBasedOnEmail true if create new user based on committer's email.
      * @return {@link User}
      */
+    @Obsolete
     public User findOrCreateUser(String csAuthor, String csAuthorEmail, boolean createAccountBasedOnEmail) {
+        return findOrCreateUser(csAuthor, csAuthorEmail, createAccountBasedOnEmail, false);
+    }
+    /**
```

Comment:
Style minor item.  Could you insert a blank line here?  That fits with the Netbeans automated formatting style that I've been preferring to use in other new additions to the plugin.

Reply:
Done